### DISCUSSION TITLE
Rust Future & Pin | SF-Zhou's Blog

### DISCUSSION THREAD

**Discussion by giscus** [SF-Zhou/sf-zhou.github.io#216] [Announcements]
# Rust Future & Pin | SF-Zhou's Blog

https://sf-zhou.github.io/rust/rust_future_and_pin.html

**Comment by SF-Zhou** [comment]
另一个 Future 的[样例](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=077875e21b93e620f0576992d0fa1c69)
